### DISCUSSION TITLE
Why is a Border Router Necessary in Thread Protocol Instead of Direct Device Access via Public IPv6 Addresses?

### DISCUSSION THREAD

**Discussion by Nuclear6** [openthread/openthread#9423] [Q&A] [answered]
In the context of the Thread open-source project, I'm curious about why a border router is required for communication instead of allowing direct device access via public IPv6 addresses. Since Thread is designed based on IPv6, what are the technical reasons behind this necessity? I'd like to understand the rationale and implications of having a border router in the Thread network. Thank you!

**Comment by jwhui** [comment, marked as answer]
Thread communicates IPv6 packets over an IEEE 802.15.4 PHY. A Thread Border Router is required to connect Thread networks to other IP-based networks like Wi-Fi, Ethernet, DOCSIS, cellular, etc.

**Reply by Nuclear6** [reply]
Got it ！I also want to ask another question, What Impact Does iPhone 15's Support for Thread Have on Previous Network?

**Reply by jwhui** [reply]
Without any details on iPhone's implementation, one can only speculate.
